ilc/server/tailor/server-router.js

```diff
@@ -59,7 +60,7 @@ module.exports = class ServerRouter {
             url.searchParams.append('routerProps', Buffer.from(JSON.stringify(reqProps)).toString('base64'));
 
             if (slotData.props !== undefined || appInfo.props !== undefined) {
-                const appProps =  _.merge({}, appInfo.props, slotData.props);
+                const appProps = _.merge({}, appInfo.props, slotData.props);
```

Comment:
suggestion - let's use deepmerge everywhere to have codebase more consistency

Reply:
It is a good point. I guess it is not a part of this PR. I am going to change it in another PR and find other places to reach consistency.